### DISCUSSION TITLE
[Feature]: Example of select-like combobox

### DISCUSSION THREAD

**Discussion by DamianGlowala** [unovue/reka-ui#1338] [Example Request]
### Describe the feature

As Radix Vue adheres to the WAI-ARIA design patterns, could we have an example added of a select-like combobox, please? This would be a perfect replacement for a select component which won't support virtualization for large datasets.

https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-select-only

### Additional information

- [ ] I intend to submit a PR for this feature.
- [ ] I have already implemented and/or tested this feature.

**Comment by zernonia** [comment]
I believe this can be easily done by `Listbox` component 😁  
Can we move this to [Example request](https://github.com/radix-vue/radix-vue/discussions/categories/example-request) discussion?

**Comment by DamianGlowala** [comment]
For sure, let's move it there 😄 I'm just not sure how listbox component (which only renders a list) could be correctly combined with a trigger (keeping in mind good accessiblity). An "official" example would be definitely appreciated! 🙏

**Comment by Perdolique** [comment]
@zernonia yep, it's a kind of `Combobox`, but without an `input`. Like a `Select` that allows for fully customised items with custom content instead of just text.
